apps/vc-api/src/vc-api/exchanges/types/exchange-verification-result.ts

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2021 - 2023 Energy Web Foundation
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * A response object from verification of a credential or a presentation.
+ * https://w3c-ccg.github.io/vc-api/verifier.html
+ */
+
+export interface VerificationResult {
```

Comment:
Interface name should match the file name
```suggestion
export interface ExchangeVerificationResult {
```